src/utils/UseWindowDimensions.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+import { debounce } from 'lodash';
+
+const useWindowDimensions = () => {
+  const { innerWidth: width, innerHeight: height } = window;
+  return { width, height };
+};
+
+export function useWindowSize(delay = 0) {

```

Comment:
Move this to lib/hooks file

Reply:
export const useWindowSize

Reply:
Replace loadsh with manual utils functions, use https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore